Libraries/LibWeb/ServiceWorker/CacheStorage.cpp

```diff
@@ -30,4 +30,10 @@ GC::Ref<WebIDL::Promise> CacheStorage::open(String const&)
     return WebIDL::create_rejected_promise(realm(), WebIDL::NotSupportedError::create(realm(), "CacheStorage.open() is not yet implemented"_string));
 }
 
+// https://w3c.github.io/ServiceWorker/#cache-storage-has
+GC::Ref<WebIDL::Promise> CacheStorage::has(String const&)
+{
+    return WebIDL::create_rejected_promise(realm(), JS::Value(false));
```

Comment:
I think we should dbgln a FIXME or something here, otherwise it might be a bunch harder to find this breaking something on a live site